doc/installation.rst

```diff
@@ -38,75 +48,59 @@ You might need to install the ``curl`` program first, if not already installed.
 
   apt-get install curl
 
-3. Install the pre-built binary packages and its dependencies by running the following commands
+3. Install the pre-built binary packages and their dependencies by running the following commands
 
 ::
 
   apt-get update
   apt-get install eos
 
-4. On older version of Ubuntu, you might need to update your installation of the Matplotlib Python package.
-   We recommend to install the most up-to-date Python package with from the Python Package Index, using
+4. On older versions of Ubuntu, you might need to update your installation of the Matplotlib Python package.
+   We recommend installing the most up-to-date Python package from the Python Package Index, using
 
 ::
 
   pip3 install --user h5py matplotlib numpy pypmc pyyaml scipy
 
 
+
+*****
 macOS
-=====
+*****
+
+Installation with `brew`
+========================
 
-The most recent development version of EOS (and all of its dependencies) can be installed automatically using `Homebrew <https://brew.sh/>`_:
+EOS (and all of its dependencies) can be installed automatically using `Homebrew <https://brew.sh/>`_:
 
 ::
 
   brew tap eos/eos
   brew install --HEAD eos
 
-We recommend to install the most up-to-date Python package with from the Python Package Index, using
+We recommend installing the most up-to-date Python package from the Python Package Index, using
 
 ::
 
   pip3 install --user h5py matplotlib numpy pypmc pyyaml scipy
 
-***********************************************
-Installing with the Python Package Index (PyPI)
-***********************************************
 
-Linux
-=====
-
-Pre-built binary packages of the most recent release of EOS are available through the `Python Package Index <https://pypi.org/>`_.
-We are presently building these binary packages, also known as wheels, for the ``manylinux2014`` platform.
-We assume that you have access to the ``pip3`` command.
-If not, you need to install ``pip3`` through via your package manager, or ask your system administrator to install it.
-
-EOS can be installed by running the following command:
 
-::
-
-  pip3 install --user eoshep
-
-
-MacOS X
-=======
-
-We are presently not building Python wheels for the MacOS platform.
-
-**********************
-Installing from Source
-**********************
+*****************************
+Build and Install from Source
+*****************************
 
 The following instructions explain how to install EOS from source on a Linux-based operating system,
-such as Debian or Ubuntu, or MacOS X.
-Other flavors of Linux or UNIX-like operating systems will likely work as well.
+such as Debian, Ubuntu, or MacOS X.
```

Comment:
macOS is not a Linux-based operating system.

Reply:
"The following instructions explain how to install EOS from source on a Linux-based operating system, such as Debian or Ubuntu, and macOS."